lib/createRoom.tsx

```diff
@@ -1,12 +1,18 @@
 import getAiTitle from "./getAiTitle";
 
-const createRoom = async (account_id: string, content: string) => {
+const createRoom = async (account_id: string, content: string, artist_id?: string) => {
   try {
     const title = await getAiTitle(content);
     const topic = title.replaceAll(`\"`, "");
-    const response = await fetch(
-      `/api/room/create?account_id=${account_id}&topic=${encodeURIComponent(topic)}`,
-    );
+    const url = new URL(`/api/room/create`, window.location.origin);
```

Comment:
Is `new URL` required here?

- before: just `fetch` from the `relative path`
- after: `new URL()`
- question: what `blocker` or `error` caused us to need to use `new URL()`?